assets/csidriveroperators/openstack-cinder/hypershift/mgmt/deployment.patch.yaml

```diff
@@ -45,25 +45,29 @@ spec:
           operator: Equal
           value: ${CONTROLPLANE_NAMESPACE}
       containers:
-      - name: openstack-cinder-csi-driver-operator
-        env:
-          - name: DRIVER_CONTROL_PLANE_IMAGE
-            value: ${DRIVER_CONTROL_PLANE_IMAGE}
-          - name: LIVENESS_PROBE_CONTROL_PLANE_IMAGE
-            value: ${LIVENESS_PROBE_CONTROL_PLANE_IMAGE}
-        volumeMounts:
-          - mountPath: /etc/guest-kubeconfig
-            name: guest-kubeconfig
-        terminationMessagePolicy: FallbackToLogsOnError
+        - name: openstack-cinder-csi-driver-operator
+          env:
+            - name: DRIVER_CONTROL_PLANE_IMAGE
+              value: ${DRIVER_CONTROL_PLANE_IMAGE}
+            - name: LIVENESS_PROBE_CONTROL_PLANE_IMAGE
+              value: ${LIVENESS_PROBE_CONTROL_PLANE_IMAGE}
+          volumeMounts:
+            - mountPath: /etc/guest-kubeconfig
+              name: guest-kubeconfig
+          terminationMessagePolicy: FallbackToLogsOnError
```

Comment:
nit: was there anything wrong with indentation here btw?

Reply:
Everything else in this file indented list items except here and line 67-68 below. It's a total nit though: I'm happy to remove it it makes this tougher to review.